ocaml/src/tools/etcd.ml

```diff
@@ -0,0 +1,48 @@
+(*
```

Comment:
That raises the question: do we want Arakoon to be the supplier of etcd functionality ?
I guess it's better to have an independent library for this.